mimic/model/nova_objects.py

```diff
@@ -515,21 +551,77 @@ def request_ips(self, http_get_ips_request, server_id):
         return dumps({"addresses": server.addresses_json()})
 
     def request_list(self, http_get_request, include_details, absolutize_url,
-                     name=u""):
+                     name=u"", limit=None, marker=None):
         """
         Request the list JSON for all servers.
 
         Note: only supports filtering by name right now, but will need to
         support more going forward.
-        """
-        return dumps(
-            {"servers": [
+
+        Pagination behavior verified against Rackspace Nova as of 2015-04-29.
```

Comment:
Minor nitpick; this should probably be "Rackspace Cloud Servers" rather than "Rackspace Nova"; "Nova", as an openstack thing, sort of implicitly means "not Rackspace specific".